package.json

```diff
@@ -165,6 +165,8 @@
     "socket.io": "^2.1.1",
     "socket.io-client": "^2.3.0",
     "svelte": "^3.24.0",
+    "svelte-json-tree": "^0.1.0",
```

Comment:
Let’s remove `svelte-json-tree` and go with `svelte-json-tree-auto` until https://github.com/tanhauhau/svelte-json-tree/issues/8 gets addressed. (Thanks for creating that fork!)